### DISCUSSION TITLE
IBM i: Refresh Connections doesn't seem to work

### DISCUSSION THREAD

**Discussion by jwoehr** [codefori/vscode-ibmi#1087] [Q&A] [answered]
Save us time! Click 'Report an Issue' from the Help and Support panel inside of VS Code after you have connected to a system.

**Describe the bug**
Closed laptop, lost VPN connection, so lost Code for i connection.
Reconnected to VPN and tried `IBM i: Refresh Connections` but that didn't seem to work.
Maybe that's not the right command.
Need a command to reconnect when connection dropped for extraneous reason.

**Comment by worksofliam** [comment, marked as answer]
Enable this in the settings:

<img width="940" alt="image" src="https://user-images.githubusercontent.com/3708366/220693630-e7f8f011-6e14-45f0-a577-ca3f4b6c6d74.png">

Restart VS Code, reconnect, then these new buttons will appear. The one on the right is reconnect.

edit: converting this to Q&A forum post.

**Reply by jwoehr** [reply]
Thanks, Liam.
